internal/config/profile.go

```diff
@@ -141,6 +169,10 @@ func (p *Profile) Service() string {
 	return settings[service]
 }
 
+func IsCloud() bool {
```

Comment:
[q] Where are you planning on using this func?

Reply:
While testing some cloud gov things I spotted a couple of places where we don't account for it and may be bundleing into ops manager/cloud manager, search for `config.Service() != cloud`